docs/tutorials/tutorial-0-Linux.rst

```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+===============================================
+
+In this tutorial, you'll take a really simple "Hello, world!" program written in
+Python, convert it into a working Ubuntu Linux project.
+
+Setup
+-----
+
+This tutorial assumes you've read and followed the instructions in
+:doc:`/intro/getting-started`. This Tutorial is for Linux Users developing for Ubuntu 16.04!
```

Comment:
There's a flow problem here - this sentence doesn't lead into the bullet points below.